infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -502,6 +502,14 @@ public void warnFlagDeprecation(final String oldFlag, final String newFlag) {
         Color.YELLOW);
   }
 
+  public void warnIgnoringWeakSubjectivityPeriod() {
```

Comment:
In an ideal world we'd be able to show this function is called when needed, but i had a quick look at beaconChainController and it might be more trouble than it's worth, so just tech debt for us to consider in future maybe...